docs(cli): deprecate Kilo Console (#12701)
Kilo-Org/kilocode: packages/kilo-docs/pages/contributing/architecture/cli-runtime.md

## Command entry points

{% callout type="warning" title="Kilo Console is deprecated" %}

> The `kilo console` command and the browser interface described on this page are deprecated. They remain documented here while the implementation is still present.

Author: ```suggestion
The `kilo console` command and the browser interface described on this page are deprecated and will be removed in an upcoming release.
```

## Kilo Console

> **Deprecated.** The Kilo Console browser interface and its `kilo console` launcher may be removed in a future release.

Author: ```suggestion
**Deprecated.** The Kilo Console browser interface and its `kilo console` launcher will be removed in a future release.
```
